package.json

```diff
@@ -51,7 +51,7 @@
     "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "babel": "^5.8.21",
+    "babel": "^5.8",
```

Comment:
Doing an `npm install` does result in different versions getting installed in these two scenarios, maybe it depends on the version of `npm` used